src/test/java/Homework18_PB.java

```diff
@@ -14,6 +20,36 @@ public class Homework18_PB extends BaseTest{
     12. Create a pull request.*/
 
 
-    
+    @Test
+    public void playSongTest() throws InterruptedException{
+
+        navigateToPage();
+        login("pooja.bankar@testpro.io", "te$t$tudent");
+        clickSubmit();
+        clickAllSongsTab();
+        selectAndPlaySong();
+        isSongPlaying();
+    }
+
+  //Locate All Songs
+           public void clickAllSongsTab() throws InterruptedException {
+               WebElement allSongsTabField = driver.findElement(By.cssSelector("a.songs"));
+                allSongsTabField.click();
+                Thread.sleep(3000);
+       }
+
+        // Locate first song from All Songs List,double click and play
+        public void selectAndPlaySong() throws InterruptedException{
+
+            Actions actions = new Actions(driver);
+            WebElement selectFirstSongField = driver.findElement(By.xpath("//*[@id='songsWrapper']//div[1]/table/tr[1]/td[2]"));
+            actions.doubleClick(selectFirstSongField).build().perform();
+           Thread.sleep(3000);
+       }
 
+       //Validate song is playing
+        public void isSongPlaying(){
+         WebElement soundbar = driver.findElement(By.xpath("//div[@data-testid='sound-bar-play']"));
+            Assert.assertTrue(soundbar.isDisplayed());
```

Comment:
**How can I beautify code in IntelliJ:**

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)